src/main/lws/lws-service.js

```diff
@@ -7,13 +7,256 @@ import { IdAttribute } from '../identity/id-attribute';
 import fetch from 'node-fetch';
 import { URLSearchParams } from 'url';
 import ethUtil from 'ethereumjs-util';
+import fs from 'fs';
+import https from 'https';
+import child_process from 'child_process';
+import sudo from 'sudo-prompt';
+import common from 'common/utils/common';
+import powerShell from 'node-powershell';
 
-export const WS_ORIGINS_WHITELIST = ['chrome-extension://knldjmfmopnpolahpmmgbagdohdnhkik'];
+export let ps = new powerShell({
+	executionPolicy: 'Bypass',
+	noProfile: true
+});
+
+export const WS_ORIGINS_WHITELIST = [
+	'chrome-extension://knldjmfmopnpolahpmmgbagdohdnhkik',
+	'chrome-extension://pfdhoblngboilpfeibdedpjgfnlcodoo'
+];
 export const WS_IP_WHITELIST = ['127.0.0.1', '::1'];
-export const WS_PORT = process.env.LWS_WS_PORT || 8898;
+export let WS_PORT = process.env.LWS_WS_PORT || 8898;
 
 const log = new Logger('LWSService');
 
+const userDataPath = common.getUserDataPath();
+
+function init() {
+	return new Promise((resolve, reject) => {
```

Comment:
I believe this should be in a separate file `lws-cert-service.js`

Reply:
yeah, I think this file is doing too much things. We should separate into small modules.